cmake/CMakeUninstall.cmake

```diff
@@ -0,0 +1,57 @@
+# xoreos - A reimplementation of BioWare's Aurora engine
```

Comment:
You need to add that file into EXTRA_DIST in the top-most rules.mk (where the other CMake files are listed).